CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Release Notes
 
+## 2.8.0
```

Comment:
Could we add a line "feat: enable displaying verification as tests when using the latest Dafny version" at the start?